### ISSUE TITLE
Store Widget shortcuts further down in paths like Tags or Widgets

### ISSUE THREAD

**Issue by dettwild** [marcelveldt/script.skin.helper.widgets#4]
Hi Marcel
I am not sure if this is the right place to mention this, given that I don't even know if this is an 'issue' or rather a 'feature request' and if this is really part of the skin helper widgets script or perhaps rather part of skin shortcuts. 
Therefore please forgive me in case I am at the wrong place and kindly point to the correct place.
As even my title may be confusing, please allow me to explain what I am after in more detail using a specific example:
I am trying to use the 'Next Episodes' as Widget, but not on the entire TV Show library, but rather filtered (using a Tag in my case,  but could also be a Genre in principle). 
When I manually navigate down the path using the Skin Helper Service Widgets plugin, I can to to 'TV Shows' - 'Tags', then select the respective Tag I want (acting as filter) and below that use the 'Next Episodes' node to end up seeing exactly what I want. Having seen this already got me excited as I figured I could also assign this as source of a widget.
However, when trying to set up the widget accordingly (btw. I am using the latest Titan Krypton Beta Skin), I can only go as far as 'TV Shows' - ' Tags' and then the Widget assignment wizard exits and in the respective skinshortcuts properties file I see the corresponding widget path as `plugin://script.skin.helper.widgets/?action=tags&mediatype=tvshows....`
I guess the latest by now you get where I am getting to: The resulting widget lists my TV Show tags and if I select one from the Widget I can continue to navigate down to 'Next Episodes', but of course I would like those next episodes (only those with the selected Tag) to show up directly in the widget.

Does that sound doable? Is there perhaps already a clever solution or workaround for that?

I am aware, that I could use custom playlists or video nodes for simple widgets like 'In progress Movies'. However, I believe the nature of 'Next Episodes' to be quite tricky and therefore don't expect that to work there.

In any case, thanks for all your great work and thanks for any hints regarding 
the topic here.

Regards
Daniel

**Comment by marcelveldt** [comment]
Hi Daniel, I was not aware of this issue. I'll look into it.

**Comment by dettwild** [comment]
Hi Marcel

Thank you very much for having a look. In the meantime I realized that the script itself does allow tag filtering and I was able to manually edit the path in the skinshortcuts  configfile to: 
`plugin://script.skin.helper.widgets/?action=recent&mediatype=episodes&tag=myTagName&reload=$INFO[Window(Home).Property(widgetreload)]`
After this manual edit, the Widget did correctly only show the episodes of Shows with the respective Tag.
I therefore conclude that the "Issue" rather lies in the assignment of the path, but I am not sure if that is the Skin itself (am am using your Titan.kyptonbeta latest release) or perhaps the Skinshortcuts plugin.
Hope that helps as it would surely be great to be able to use the Skin Settings Gui to set up the Widgets ;-).
Cheers, Daniel
